packages/spatial/src/physics/systems/PhysicsSystem.test.ts

```diff
@@ -298,6 +297,7 @@ describe('PhysicsSystem', () => {
 
       const physicsSystemReactor = SystemDefinitions.get(PhysicsSystem)?.reactor
 
+      /** @todo Why is the world not recreated as expected ?? */
```

Comment:
i tested this myself and it seems to be fine. what error are you seeing?

Reply:
It's a retroactive note.
I saw the `.skip` but I never let skips in without a todo, so added it.

If I remove the `.skip` from the test right below this line, I get this error:
```ts
AssertionError: Got unwanted exception.
Actual message: "Physics world not found"
 ❯ src/physics/systems/PhysicsSystem.test.ts:310:16
    308|         root.run()
    309|         assert.equal(hasComponent(physicsWorldEntity, SceneComponent), true)
    310|         assert.doesNotThrow(() => Physics.destroyWorld(getComponent(physicsWorldEntity, UUIDComponent)))
       |                ^
    311|       })
    312|     }) //:: PhysicsSceneReactor
```